Makefile

```diff
@@ -2,7 +2,7 @@ REGISTRY?=gcr.io/k8s-staging-prometheus-adapter
 IMAGE=prometheus-adapter
 ARCH?=$(shell go env GOARCH)
 ALL_ARCH=amd64 arm arm64 ppc64le s390x
-GOPATH:=$(shell go env GOPATH)
+GOPATH:=/opt/homebrew
```

Comment:
remove before merge -> my homebrew file is in a different location than where this command points to